dashboard/README.md

```diff
@@ -16,3 +16,13 @@ Run the dev server with
 ```sh
 pnpm dev
 ```
+
+# Routing and State Management
+
+A big part of this project is to have shareable links
+So we should use the URL to manage state as much as possible.
+
+Also, we are using file based routing in the tanstack router, only files that starts with ~ are read by the Vite plugin.
+
+
+
```

Comment:
nit: left only one extra line

Reply:
fixed in
https://github.com/kernelci/dashboard/pull/93/commits/d1fd009e2c70dd340f5e557bc17ea6cf2b971c1d